ui/components/app/transaction-decoding/components/ui/accreditation/accreditation.component.js

```diff
@@ -41,15 +41,37 @@ const Accreditation = ({ fetchVia, address }) => {
           rel="noopener noreferrer"
           title={t('etherscanView')}
         >
-          {fetchVia}
+          Etherscan
         </Button>
-        <Typography variant={TYPOGRAPHY.H7} boxProps={{ margin: 0 }}>
-          {t('transactionDecodingAccreditationDecoded')}
-        </Typography>
       </>
     );
   };
 
+  const TruffleNotice = () => {
+    return (
+      <Typography variant={TYPOGRAPHY.H7} boxProps={{ margin: 0 }}>
+        {t('transactionDecodingAccreditationDecoded')}
+      </Typography>
+    );
+  };
+
+  const AccreditationLink = () => {
+    switch (fetchVia) {
```

Comment:
For defensive coding, we may want to lowercase this (in case server responds `"Etherscan"` or some variation)